functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ChangeApplier.java

```diff
@@ -111,7 +111,9 @@ private void applyStashChangesLocally(List<RevCommit> stashesToApply) throws Git
         // So we create temporary commits to overcome this issue
         // and reset softly on the way to have it all as local changes
         for (final RevCommit stash : stashesToApply) {
-            git.stashApply().setStashRef(stash.getName()).call();
+            if(stash != null) {
```

Comment:
I think this check shouldn't be here - maybe it's better to change this line

```java
stashesToApply.add(git.stashCreate().setIncludeUntracked(true).call());
```

in `applyLocallyFromTags` instead to prevent from adding `null`s?